sphinx/templates/latex/content.tex_t

```diff
@@ -16,7 +16,9 @@
 <%= passoptionstopackages %>
 <%= geometry %>
 <%= inputenc %>
+<% if latex_engine == 'pdflatex' -%>
 <%= utf8extra %>
+<% endif -%>
```

Comment:
actually I need utf8extra also for unicode engines after having rebased on master, hence now it is inputenc which is loaded only if pdflatex engine is used. OK for Japanese ?